ckanext/pages/actions.py

```diff
@@ -14,6 +18,20 @@
 
 
 import db
+
+
+def image_mime_type_validator(file_path):
+    allowed_types = [
+        'image/jpeg',
+        'image/bmp',
+        'image/gif',
+        'image/png'
+    ]
+    file_type, _ = mimetypes.guess_type(file_path)
```

Comment:
Does it check file contents too? E.g. what will it return for a text file with '.bmp' extension?